lib/graphql/mutations.js

```diff
@@ -20,7 +21,7 @@ export const createUserQuery = gql`
   }
 `;
 
-const createCollectiveQuery = gql`
+const createCollectiveQuery = gqlV2`
```

Comment:
If you modify this queries in place, this is gonna impact the old flow. I suggest to add the mutations directly where you need them.